### PR TITLE
Add aniso8601.

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

File: recipes/aniso8601/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - python-dateutil
```
Comment (ocefpaf) [inline recipes/aniso8601/meta.yaml]: It is a minor thing but can you add a line here to separate the fields.

File: recipes/aniso8601/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+test:
+  imports:
+    - aniso8601
+
+about:
+  home: https://bitbucket.org/nielsenb/aniso8601
+  license: BSD-3
```
Comment (ocefpaf) [inline recipes/aniso8601/meta.yaml]: Another minor thing: We usually write `BSD 3-Clause`

File: recipes/aniso8601/meta.yaml (hunk 1 above)
Comment (vamega) [inline recipes/aniso8601/meta.yaml]: Will do.

File: recipes/aniso8601/meta.yaml (hunk 2 above)
Comment (vamega) [inline recipes/aniso8601/meta.yaml]: Will update.

**Pull request merged**